packages/exo/src/exo-makers.js

```diff
@@ -186,8 +218,8 @@ export const defineExoClassKit = (
     // Be careful not to freeze the state record
     const state = seal(init(...args));
     // Don't freeze context until we add facets
-    /** @type {KitContext<ReturnType<I>,F>} */
-    const context = { state, facets: {} };
+    /** @type {{ state: ReturnType<I>, facets: unknown }} */
+    const context = { state, facets: null };
```

Comment:
Why is null here correct? The original
```suggestion
    const context = { state, facets: {} };
```
seems more correct.